src/main/java/de/minebench/syncinv/messenger/Message.java

```diff
@@ -68,10 +68,10 @@ public Object read() {
 
     /**
      * Generate a byte array out of the data of this message
-     * @return          The generated byte array (starts with the sender
-     *                  then the type ordinal, then the amount
-     *                  of data being send and each data object);
-     *                  an empty one if an error occurred
+     * @return The generated byte array (starts with the sender
+     * then the type ordinal, then the amount
+     * of data being send and each data object);
+     * an empty one if an error occurred
```

Comment:
This looks weird. Additional lines should probably be indented. (And line breaks moved too if whitespace at the start is removed as that leaves more room at the end of the line for text)

Reply:
Well, in this case we should follow a standard everywhere in the plugin instead of random indentation. Changed the other occurrence of additional lines before my changes too.